### PR TITLE
Pcm 1974

### PR DESCRIPTION
**Pull request by maxwellmooney13**

### CODE REVIEW

File: react-app/src/library/services/vendor-implementations/jabra/jabra.ts [hunk 1]
```diff
@@ -201,7 +201,22 @@ export default class JabraService extends VendorImplementation {
...
-  async answerCall (): Promise<void> {
+  async answerCall (conversationId: string, autoAnswer?: boolean): Promise<void> {
+    if (autoAnswer) {
+      this.pendingConversationId = conversationId;
+      try {
+        this.callLock = await this.callControl.takeCallLock();
+      } catch ({ message, type }) {
+        if (this.checkForCallLockError(message, type)) {
+          this.logger.info(message);
+          this.callControl.ring(false);
```
Comment (jensengar) [inline react-app/src/library/services/vendor-implementations/jabra/jabra.ts:212]: do we need a return in here or can we just rely on the 
```
this.callControl.ring(false);
this.callControl.offHook(true);
```

below?

Comment (maxwellmooney13) [reply react-app/src/library/services/vendor-implementations/jabra/jabra.ts:212]: Yanno, looking at this now and some other similar approaches, I think we should be fine of just relying on the one below.

File: react-app/src/components/toggle-switch/toggle-switch.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
+.toggle-switch {
+    height: 0;
+    width: 0;
+    visibility: hidden;
```
Comment (zservies) [inline react-app/src/components/toggle-switch/toggle-switch.css:4]: NIT: This works, but in the future a simpler way to hide an element AND not get it to take up any space in the layout is to just use `display: none`. That'll remove it from the document entirely and you won't have to set a `height` and `width`.

File: react-app/src/library/services/vendor-implementations/jabra/jabra.ts [hunk 3]
```diff
@@ -235,20 +250,21 @@ export default class JabraService extends VendorImplementation {
...
       } else {
         this.logger.error(type, message);
       }
     }
+
+    if (this.callLock) {
+      this.pendingConversationId = callInfo.conversationId;
+      this.pendingConversationIsOutbound = true;
+      this.callControl.offHook(true);
+      return Promise.resolve();
```
Comment (jensengar) [inline react-app/src/library/services/vendor-implementations/jabra/jabra.ts:266]: This return is pointless

Comment (maxwellmooney13) [reply react-app/src/library/services/vendor-implementations/jabra/jabra.ts:266]: Totally right, that was a relic that I overlooked to remove

File: react-app/src/components/toggle-switch/toggle-switch.css (hunk 2 above)
Comment (maxwellmooney13) [reply react-app/src/components/toggle-switch/toggle-switch.css:4]: I always get visibility and display mixed up... thank you for the suggestion!

Comment (zservies) [reply react-app/src/components/toggle-switch/toggle-switch.css:4]: Realistically I'm overanalyzing the demo-app but I've no headsets so its about the only thing I'm really good for when it comes to reviewing these right now lol.

**Pull request merged**